hera_cal/lst_stack/calibration.py

```diff
@@ -463,6 +465,9 @@ def lstbin_absolute_calibration(
             the auto-correlations will be used to calculate the gain amplitude if they are provided.
             Note that the auto_model must also be provided if this flag is set to True and autos are
             provided. If set to False, the auto-correlations will not be used for calibration.
+        refpol : str, default='Jee'
+            The reference polarization to use for cross-polarization phase calibration. This is
+            the polarization that the other polarizations are calibrated to. The default is 'Jee'.
```

Comment:
"other polarization is" since there are only two

Reply:
Fixed